source/main/scripting/bindings/GameScriptAngelscript.cpp

```diff
@@ -100,6 +100,16 @@ void RoR::RegisterGameScript(asIScriptEngine *engine)
     result = engine->RegisterObjectMethod("GameScriptClass", "int getNumTrucks()", asMETHOD(GameScript, getNumTrucks), asCALL_THISCALL); ROR_ASSERT(result >= 0);
     result = engine->RegisterObjectMethod("GameScriptClass", "int getCurrentTruckNumber()", asMETHOD(GameScript, getCurrentTruckNumber), asCALL_THISCALL); ROR_ASSERT(result >= 0);
     result = engine->RegisterObjectMethod("GameScriptClass", "BeamClass @spawnTruck(string &in, vector3 &in, vector3 &in)", asMETHOD(GameScript, spawnTruck), asCALL_THISCALL); ROR_ASSERT(result >= 0);
+    result = engine->RegisterObjectMethod("GameScriptClass", "BeamClass @spawnTruckAI(string &in, vector3 &in, string &in, string &in)", AngelScript::asMETHOD(GameScript, spawnTruckAI), AngelScript::asCALL_THISCALL); ROR_ASSERT(result >= 0);
+    result = engine->RegisterObjectMethod("GameScriptClass", "array<vector3> @getWaypoints()", AngelScript::asMETHOD(GameScript, getWaypoints), AngelScript::asCALL_THISCALL); ROR_ASSERT(result >= 0);
+    result = engine->RegisterObjectMethod("GameScriptClass", "int getAIVehicleCount()", AngelScript::asMETHOD(GameScript, getAIVehicleCount), AngelScript::asCALL_THISCALL); ROR_ASSERT(result >= 0);

```

Comment:
All the `getAi*()` funcs either need to be added to angelscript documentation (https://github.com/RigsOfRods/rigs-of-rods/blob/master/doc/angelscript/Script2Game/GameScriptClass.h) or replaced with cvars, which really exist for that purpose:
```
// in VehicleAiPanel constructor (once per session):
App::GetConsole()->cVarCreate("ai_vehicle_count", CVAR_TYPE_INT);

//in VehicleAiPanel:
App::GetConsole()->cVarGet("ai_vehicle_count")->setVal(int(ai_vehicle_count));

// in AngelScript:
int vehicle_count = console.cVarGet("ai_vehicle_count").getInt();
```